integration_tests/configs/default.jsonnet

```diff
@@ -101,7 +103,10 @@
         feemarket: {
           params: {
             no_base_fee: false,
-            base_fee: '100000000000',
+            base_fee_change_denominator: '300',
+            elasticity_multiplier: '4',
+            base_fee: '10000000000000',
+            min_gas_price: '10000000000000'
```

Comment:
it makes it basically a constant base fee in integration test, because the block gas used ratio would be low.

Reply:
we can have a dedicated `min_gas_price` test case, test the behavior difference w/o it, if want to.

Reply:
do you mean change config only for test_eip1559.py? how about ibc related config?